### PR TITLE
refine docs, add accuracy data, add receip and eval scripts

### PR DESCRIPTION
**Pull request by WeiweiZhang1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,6 +182,10 @@ print(tokenizer.decode(model.generate(**inputs, max_new_tokens=50)[0]))
...
 |--------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| microsoft/Phi-3-vision-128k-instruct  |  [accuracy](./examples/multimodal-modeling/Phi-3-vision/README.md), [recipe](./examples/multimodal-modeling/Phi-3-vision/run_autoround.sh)
+| Qwen/Qwen-VL  |  [accuracy](./examples/multimodal-modeling/Qwen-VL/README.md), [recipe](./examples/multimodal-modeling/Qwen-VL/run_autoround.sh)
+| liuhaotian/llava-v1.5-7b  |  [accuracy](./examples/multimodal-modeling/Llava/README.md), [recipe](./examples/multimodal-modeling/Llava/run_autoround.sh)
+| microsoft/Phi-3-vision-128k-instruct  |  [accuracy](./examples/multimodal-modeling/README.md), [recipe](./examples/multimodal-modeling/run_autoround.sh)
```
Comment (wenhuach21) [inline README.md]: phi3 repeated

File: examples/multimodal-modeling/Phi-3-vision/README.md [hunk 2]
```diff
@@ -47,13 +49,72 @@ Include the flag `--adam`. Note that AdamW is less effective than sign gradient
...
+generate_ids = generate_ids[:, inputs['input_ids'].shape[1]:]
+response = processor.batch_decode(generate_ids, skip_special_tokens=True, clean_up_tokenization_spaces=False)[0] 
+
+print(response)
+
+```
+
+
+## 4. Results
+Using [COCO 2017](https://cocodataset.org/) and [LLaVA-Instruct-150K](https://huggingface.co/datasets/liuhaotian/LLaVA-Instruct-150K) datasets for quantization calibration, and lm_eval dataset for evaluation. please follow the [recipe](./run_autoround.sh) and [evaluate script](./run_eval.sh). The results for Phi-3-vision-128k-instruct are as follows:
```
Comment (wenhuach21) [inline examples/multimodal-modeling/Phi-3-vision/README.md:103]: why using these tasks to eval this model?

File: examples/multimodal-modeling/Qwen-VL/mm_evaluation/main.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+            if args.device == "hpu":
+                from auto_round.auto_quantizer import AutoHfQuantizer
+    model_name = args.model_name
+    torch_dtype = torch.float
+    if args.model_dtype != None:
+        if args.model_dtype == "float16" or args.model_dtype == "fp16":
+            torch_dtype = torch.float16
+        if args.model_dtype == "bfloat16" or args.model_dtype == "bfp16":
+            torch_dtype = torch.bfloat16
+    dtype_abd = convert_dtype_torch2str(torch_dtype)
```
Comment (wenhuach21) [inline examples/multimodal-modeling/Qwen-VL/mm_evaluation/main.py]: rename "dtype_abd"

File: README.md [hunk 4]
```diff
@@ -188,6 +188,8 @@ Please note that an asterisk (*) indicates third-party quantized models, which m
 
 Model                                | Supported                                                           |
 |--------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| microsoft/Phi-3-vision-128k-instruct  |  [recipe](./examples/multimodal-modeling/Phi-3-vision/run_autoround.sh)
```
Comment (wenhuach21) [inline README.md]: move phi-2 to the end

File: examples/multimodal-modeling/Qwen-VL/mm_evaluation/main.py [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+    import sys
+
+    sys.path.insert(0, '../../../')
+    import time
+    import torch
+    import argparse
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--model_name", default="/models/opt-125m/"
```
Comment (wenhuach21) [inline examples/multimodal-modeling/Qwen-VL/mm_evaluation/main.py:13]: change the default model name

**Review by wenhuach21 — APPROVED** [review]
